### ISSUE TITLE
"jQuery is not defined" when using the NPM installation method.

### ISSUE THREAD

**Issue by JulsWlf** [caike/jQuery-Simple-Timer#26]
I followed the NPM installation method:
```
"use strict";
let $ = require("jquery");
require("jquery-simple-timer")($);

$(() => {
  $('.timer').startTimer();
});
```

and it says jQuery is not defined.

Not sure if this has something to do with the fact that Im transpiling ES6 code using babelify and browserify.

Note: The jQuery is not defined comes from the plugin file itself "jquery.simple.timer.js:189"

Thanks in advance!

**Comment by caike** [comment]
Hmm.. maybe this is my fault since I haven't yet published the latest version of this to npm yet (#19). 
But either way, jQuery should be installed prior to using this plugin - it is *not* installed automatically. Have you tried installing jQuery first and then using the plugin straight from the [source file](https://github.com/caike/jQuery-Simple-Timer/blob/master/jquery.simple.timer.js) ?

**Comment by JulsWlf** [comment]
Yeah, I had jQuery already installed and was importing it via CommonJS method.
Turns out it is indeed an old version in npm, I downloaded the file directly from here and it works just fine!

**Comment by caike** [comment]
Cool! Sorry about that. Working on a fix for #25 and as soon as I'm done with that I'll publish the new version to npm.

**Comment by caike** [comment]
New version has been published to npm 🚀
